### PR TITLE
Матвеев Сергей - Лабораторная работа №2

### PR DESCRIPTION
**Pull request by MatveevSE1**

### CODE REVIEW

File: modules/template_binary_heap/include/template_heap.h [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    for (size_t i = 0; i < buffer.size(); i++) {
+        Push(buffer[i]);
+    }
+}
+
+template<typename T>
+void THeap<T>::Push(T value) {
+    vec.push_back(value);
+    Emersion();
+}
```
Comment (dkurt) [inline modules/template_binary_heap/include/template_heap.h:45]: Can you move implementation to .cpp?

Comment (MatveevSE1) [reply modules/template_binary_heap/include/template_heap.h:45]: No, because it uses templates

**Comment by MatveevSE1** [comment]
Check it out @AlexIK-0 @Hetzeer

**Comment by Hetzeer** [comment]
Nice job.

**Comment by AlexIK-0** [comment]
All ok!

**Pull request merged**